docs/process_manual/2020/advice.md

```diff
@@ -0,0 +1,38 @@
+# Advice for Future Process Groups
+
+This section will describe our views and experiences, and what we found out that worked. 
+Previous advice can be found [here](../../getting_started/advice_for_future_giraf/process_advice.md).
+
+## The Process in General
+
+* The project is everyone's, the process and PO group does not have more authority than everyone else, just more responsibility.
+* Take your time to understand the process and project, read the wiki, it is well spent.
+* Use the process exactly as described on this wiki and then modify it, to avoid confusion.
```

Comment:
"Use the process and then modify it"? Is it ment that they take the current process that exists, and then change it to what would fit their process?